xray/utils/downloadindexer.go

```diff
@@ -18,19 +18,19 @@ import (
 	"github.com/jfrog/jfrog-client-go/xray"
 )
 
-const indexerFileName = "indexer-app"
-
-// TODO: Should be changed back to 3.28.0 before merge
-const graphScanMinVersion = "3.0.0"
+const (
+	indexerFileName     = "indexer-app"
+	GraphScanMinVersion = "3.0.0"
```

Comment:
Looks like this const can be made private.

Reply:
I wanted to use it in the CLI tests, but may change it back for now.